lib/mollie/api/object/payment.rb

```diff
@@ -15,50 +15,103 @@ class Payment < Base
         RECURRINGTYPE_RECURRING = "recurring"
 
         attr_accessor :id,
-                      :status,
                       :mode,
-                      :amount,
-                      :description,
-                      :method,
                       :created_datetime,
+                      :status,
                       :paid_datetime,
-                      :expired_datetime,
                       :cancelled_datetime,
+                      :expired_datetime,
+                      :expiry_period,
+                      :amount,
+                      :amount_refunded,
+                      :amount_remaining,
+                      :description,
+                      :method,
+                      :metadata,
+                      :locale,
+                      :profile_id,
+                      :settlement_id,
                       :customer_id,
                       :recurring_type,
                       :mandate_id,
                       :subscription_id,
-                      :settlement_id,
-                      :metadata,
-                      :details,
-                      :links
+                      :links,
+                      :details
 
         def open?
```

Comment:
without the status_ it's already pretty clear because of the question mark. So I think we can remove it.